department-of-veterans-affairs/caseflow
# mbeard/APPEALS-32384

Resolves [APPEALS-32384](https://jira.devops.va.gov/browse/APPEALS-32384)

# Description
Created static lever component, storybook, and jest test. This component is passed in data to display the inactive levers outlined in the JIRA ticket.
## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-32384) or list them below
2. Start storybook and select the story "Static Levers"
3. Ensure that the component is rendered correctly and styling is acceptable
4. Check Lighthouse and ANDI for compliance
5. Run Jest Test for 'StaticLever' to ensure passing


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
